packages/reactivity/src/computed.ts

```diff
@@ -24,6 +25,11 @@ export interface WritableComputedOptions<T> {
   set: ComputedSetter<T>
 }
 
+export const COMPUTED_SIDE_EFFECT_WARN =
+  `[Vue warn] Computed keep dirty after evaluation.` +
```

Comment:
There's already `[Vue warn] ` in `function warn`

Reply:
good point!